CHANGELOG.md

```diff
@@ -9,11 +9,16 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### 🐞 Fixed
 - Fix channel list parsing failing completely if one channel fails parsing [#2541](https://github.com/GetStream/stream-chat-swift/pull/2541)
+- Fix `ChatChannelController` ignoring initial pagination parameter [#2546](https://github.com/GetStream/stream-chat-swift/pull/2546)
 
 ## StreamChatUI
+### 🔄 Changed
+- Deprecated `ChatMessageListVC.jumpToMessage(message:)` -> `ChatMessageListVC.jumpToMessage(id:)` [#2546](https://github.com/GetStream/stream-chat-swift/pull/2546)
```

Comment:
Given it was introduced this last release, I am not sure anyone is using it. Maybe it is not needed to add deprecations but just the entry on the changelog as "renamed". WDYT?

Reply:
Discussed internally 👍